modules/jdbc/src/main/java/org/testcontainers/jdbc/ConnectionDelegate.java

```diff
@@ -0,0 +1,18 @@
+package org.testcontainers.jdbc;
+
+import lombok.Data;
+import lombok.experimental.Delegate;
+
+import java.sql.*;
+import java.util.Map;
+import java.util.Properties;
+import java.util.concurrent.Executor;
+
+class ConnectionDelegate implements Connection {
+    @Delegate(types=Connection.class)
```

Comment:
nit: `types` here is unnecessary because it uses the variable's type, which equals to `Connection` already

Reply:
done, thx